Bio/Phylo/PhyloXML.py

```diff
@@ -593,6 +593,7 @@ def __eq__(self, other):
             value : float
                 confidence value
         """
+        
```

Comment:
That should not be a blank line according to ``pydocstyle``, this breaks its rule ``D202 No blank lines allowed after function docstring``